apps/api/src/domain/domainModelValidators/utilities/validateSimpleInvariants.ts

```diff
@@ -0,0 +1,11 @@
+import { buildSimpleValidator } from '@coscrad/validation';
```

Comment:
Nice to see this!  I do get an error here:

<img width="1016" alt="image" src="https://user-images.githubusercontent.com/63014783/168949861-a09d1335-0af0-4fc6-bc8a-bcba9d4297c8.png">

Reply:
Note: I did an F12 on `DomainModelCtor` and when I came back to the file, the red error line disappeared.  It was complaining about not having an expected argument.  Seems to me it has one.

Reply:
After pulling, I'd restart VS code. You probably have to open from the root of the monorepo for it to pick up on the lib.